src/bin/util/mod.rs

```diff
@@ -1,27 +0,0 @@
-#[cfg(unix)]
-pub(crate) mod profiling;
```

Comment:
Is it necessary to move these files?

Reply:
Yes. Otherwise, It will introduce some warnings when we only need to use a part of functions in `mod`.